src/main/java/org/embulk/gradle/embulk_plugins/VersionlessDependency.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2022 The Embulk project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.embulk.gradle.embulk_plugins;
+
+import groovy.util.Node;
+import java.util.Objects;
+import org.gradle.api.artifacts.ResolvedArtifact;
+import org.gradle.api.artifacts.component.ModuleComponentIdentifier;
+
+final class VersionlessDependency {
+    private VersionlessDependency(
+            final String group,
+            final String artifactName,
+            final String classifier) {
+        final StringBuilder builder = new StringBuilder();
+
+        if (group == null) {
+            this.group = "";
+        } else {
+            this.group = group;
+            builder.append(group);
+        }
+
+        builder.append(":");
```

Comment:
is group optional? It means stringified could be `:embulk-input-simple`.

Reply:
Yeah, it normally does not happen, but I expected it can happen in a local subproject case.